# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/moz.build
# Commit: ea88b243717a
# Full Hash: ea88b243717a1dcdc2c02a57af70685959c5d649
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add D3D11 RenderCompositor for SWGL. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95797
# ==============================================================================

diff -r c68ab3e8a465 -r ea88b243717a gfx/webrender_bindings/moz.build
--- a/gfx/webrender_bindings/moz.build	Thu Nov 05 18:27:40 2020 +0000
+++ b/gfx/webrender_bindings/moz.build	Fri Nov 06 00:00:47 2020 +0000
@@ -76,9 +76,11 @@
     EXPORTS.mozilla.webrender += [
         "DCLayerTree.h",
         "RenderCompositorANGLE.h",
+        "RenderCompositorD3D11SWGL.h",
         "RenderD3D11TextureHost.h",
     ]
     UNIFIED_SOURCES += [
+        "RenderCompositorD3D11SWGL.cpp",
         "RenderD3D11TextureHost.cpp",
     ]
     SOURCES += [
@@ -118,3 +120,4 @@
 FINAL_LIBRARY = "xul"
 
 CXXFLAGS += CONFIG["TK_CFLAGS"]
+CXXFLAGS += CONFIG["MOZ_CAIRO_CFLAGS"]